hpx/runtime/applier/trigger.hpp

```diff
@@ -10,6 +10,7 @@
 
 #include <hpx/runtime/actions/continuation.hpp>
 
+/*
```

Comment:
Those are API functions, I'd say we should leave them in. They are useful in their own right.